Pipfile

```diff
@@ -15,7 +15,7 @@ click = "*"
 geojson = "*"
 parallelpipe = "*"
 psutil = "*"
-psycopg2 = "*"
+psycopg2-binary = "*"
```

Comment:
you don't want the binary in production. You will have to add the libpg-dev library in the docker image if the build fails
https://www.psycopg.org/docs/install.html